src/components/look-controls.js

```diff
@@ -28,7 +28,11 @@ module.exports.Component = registerComponent('look-controls', {
 
   play: function () {
     this.previousPosition.set(0, 0, 0);
-    this.addEventListeners();
+
+    // defer adding listeners until everything has run.
```

Comment:
The `canvas` component will trigger `render-target-loaded` event on the scene. Want to try that?